cyences_app_for_splunk/appserver/static/cs_overview.js

```diff
@@ -50,6 +50,8 @@ require([
         { id: 'cs_device_inventory_table', title: 'Device Inventory' },
         { id: 'cs_user_inventory_table', title: 'User Inventory' },
         { id: 'cs_malicious_ip_list', title: 'Malicious IP List' },
+        { id: 'cs_mssql', title: 'MSSQL' },
+        { id: 'cs_oracle', title: 'Oracle' },
     ]
 
     let panel_depends_tokens = [
```

Comment:
once you create the panel add the its depends token and associated products (MSSQL & Oracle) in this list to show/hide panel.

Reply:
I think it will work by default based on the nav.xml. Are you sure?

Reply:
Yes, nav.xml is to show/hide the dashboard links available in top panel and navigation menu.

panel_depends_tokens is to show/hide the overview dashboard's panels which displays the alerts. (this change I did later.)